### PR TITLE
Create animals table

### PR DESCRIPTION
**Pull request by BonkeGcobo**
What we did in this pull Request::
 1) Created animals table
  2) Insert data into the animals table
   3) Query data from the table

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -1,5 +1,8 @@
 /* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
+INSERT INTO animals(name, date_of_birth, neutered, weight_kg) values ('Agumon', '2020-02-03', TRUE, 10.23);
```
Comment (uwadonat) [inline data.sql]: - [x] Let's add `escape_attemps` field in this query.

File: data.sql [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+INSERT INTO animals(name, date_of_birth,escape_attemps ,neutered, weight_kg) values ('Gabumon', '2018-11-15',2, TRUE, 8);
+INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attemps) values('Pikachu', '2021-01-07',15.04, FALSE, 1);
+INSERT INTO animals(name, date_of_birth, weight_kg, neutered, escape_attemps) values ('Devimon', '2021-05-12',11, TRUE, 5);
+/*INSERT INTO animals (name) VALUES ('Luna');
 INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name) VALUES ('Charlie');*/
```
Comment (uwadonat) [inline data.sql]: - [x] remove this commented lines to make your code more clean

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **@[BonkeGcobo](https://github.com/BonkeGcobo),** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your Pull request  message and Readme file are very descriptive ✔️
- Good commit messages ✔️
almost

## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @BonkeGcobo 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- You documented your work professionally.✔
- Github inline comments feature is used properly.
- You asked for the changes in a polite and friendly way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**